components/GameCard/GameCard.js

```diff
@@ -1,16 +1,31 @@
 import styled from "styled-components";
-import { BsPersonCircle } from "react-icons/bs";
+import { BsPersonCircle, BsX } from "react-icons/bs";
+import DeleteButton from "../DeleteButton";
 
-export default function GameCard({ type, name, date, time, place, court }) {
+export default function GameCard({
+  type,
+  name,
+  date,
+  time,
+  place,
+  court,
+  deleteCard,
+  id,
```

Comment:
do you need the `id`?

Reply:
I think so... how the id prop is used by the onClick on the deleteCardContainer